app/src/main/java/helper/CustomTextView.java

```diff
@@ -0,0 +1,29 @@
+package helper;
+
+import android.annotation.SuppressLint;
+import android.content.Context;
+import android.text.Html;
+import android.util.AttributeSet;
+import android.util.Log;
+import android.support.v7.widget.AppCompatTextView;
+
+
+public class CustomTextView extends AppCompatTextView {
+
+    public CustomTextView(Context context) {
+        super(context);
+    }
+
+    public CustomTextView(Context context, AttributeSet attrs) {
+        super(context, attrs);
+    }
+
+    public CustomTextView(Context context, AttributeSet attrs, int defStyleAttr) {
+        super(context, attrs, defStyleAttr);
+    }
+
+    @Override
+    public void setText(CharSequence text, BufferType type) {
+        super.setText(Html.fromHtml(text.toString()), type);
```

Comment:
Add a version check here as mentioned in [this answer](https://stackoverflow.com/a/37905107) in order to maintain version compatibility.